### ISSUE TITLE
Slides Screen :: Status Messages :: When Packages From ISO Are Used, Status Should Say So

### ISSUE THREAD

**Issue by lots0logs** [Antergos/Cnchi#776]
### Problem:
Currently the status messages are the same regardless of whether a package is being downloaded from the repos or being copied from the ISO's cache. We should change the messages so that it's obvious to the user which packages are actually being downloaded.

**Comment by karasu** [comment]
Are you sure about this? I think that the log shows if a package is not being downloaded because it's found in the ISO cache and copied from there.

**Comment by lots0logs** [comment]
Yeah. There is a wide-spread myth concerning Cnchi that gets repeated allover the internet to the point that people think its true (which is that cnchi always downloads from repos no matter what). Then people complain and blame us for "not using cached packages on the ISO" when in all actuality we have been doing it for years.

**Comment by Noxbru** [comment]
@lots0logs Wait, what? Since when is that true? and in which cases does Cnchi not download the required packages? Does that apply to both the Live ISO and the Minimal ISO?

Honestly, if you want us to stop saying that Cnchi always downloads everything, you should inform us that we are wrong and correct us.

**Comment by karasu** [comment]
@Noxbru 

Cnchi downloads ONLY those packages that are not cached in the ISO OR if there is a newer version online (compared to the one that's in the iso).

You'll always end with the latest version of each package (either from the ISO or downloaded).

We've been doing it this way for ages, as @lots0logs confirmed.

In minimal ISO this does not apply as we delete all the cached packages from the ISO, so cnchi has to download everything.

I hope this clarifies it 😀

**Comment by Noxbru** [comment]
@karasu 

So, there are MORE differences between the minimal and the Live ISOs than just the environment. I think that we all believed that the only difference was the DE and the set of programs already installed.


> You'll always end with the latest version of each package (either from the ISO or downloaded).

Yes, we know that we end with the latest version. But we assumed that because of that (and of not having a cache) that cnchi had to download everything always.

> We've been doing it this way for ages, as @lots0logs confirmed.

And yet, I discover this now, and I've been using Antergos for years. And I'm pretty sure that others don't even know about this.

Are we 100% sure that using the Live ISO and the minimal one we would install the same packages and that there won't be any difference between the ones in the repos and the ones cached?

**Comment by karasu** [comment]
>So, there are MORE differences between the minimal and the Live ISOs than just the environment. I think that we all believed that the only difference was the DE and the set of programs already installed.

Yes. If we said otherwise at some point I'm sorry, but I do not recall saying nothing of the like.

>Yes, we know that we end with the latest version. But we assumed that because of that (and of not having a cache) that cnchi had to download everything always.

But normal ISO has cache. That's why it's soooo big (nearly 2GB).

>And yet, I discover this now, and I've been using Antergos for years. And I'm pretty sure that others don't even know about this.

We do not have the time or the resources to publish everything we do/change. I'll try to update our website with this information.

>Are we 100% sure that using the Live ISO and the minimal one we would install the same packages 
Yes, we are.

>and that there won't be any difference between the ones in the repos and the ones cached?
I do not know if I follow you here. Of course there will be differences in packages that are not the same version. If they are the same version, they will be the same package. As simple as that.

**Comment by Noxbru** [comment]
> Yes. If we said otherwise at some point I'm sorry, but I do not recall saying nothing of the like.

Other possibility is that it was never said and that we had to guess with what we had. For example, in the web it says 'A fully functional GNOME Live Desktop Environment', and I thought that the size difference was mostly due to this. And also, that it doesn't tell otherwise.

> But normal ISO has cache. That's why it's soooo big (nearly 2GB).

As said, now I discover that. (BTW, you have to update the info in the 'Try it' page, it says 1.7 G)

> We do not have the time or the resources to publish everything we do/change.

Of course, and I understand that you don't. But in this case, as @lots0logs has pointed out, and I quote "There is a wide-spread myth concerning Cnchi that gets repeated allover the internet to the point that people think its true (which is that cnchi always downloads from repos no matter what)." And yet you haven't informed us that that is false. Or at least I haven't seen it until now.


BTW, yesterday I was able to install a VM with the minimal ISO and today it is failing with the Live one, same choices.

**Comment by lots0logs** [comment]
> And yet you haven't informed us that that is false.

That is exactly why I opened this issue.

**Comment by Noxbru** [comment]
@lots0logs 

I'm going to ask you a very serious question: How are the people who don't usually install antergos or who does but does so in kind of an 'automatic' way and latter has to help users with their problems supposed to know about this?

**Comment by lots0logs** [comment]
That is not the expectation. Besides, this is not something that requires anyone to do or even know anything. We will make it so that the UI clearly informs the user what is going on, Its as simple as that.
